README.md

```diff
@@ -643,6 +643,27 @@ $ node -e "console.log(require('./index).main)"
 [Function]
 ```
 
+If you want to be able access modules and their functions from JavaScript land,
+(highly useful for use in `<script>` tags in the browser), then you can use
+a recipe like the following. In your index.js that is input the the bundler
+(e.g. Parcel):
```

Comment:
I'm a bit confused by this, because just in the next section we talk about using the bundle from JS, so maybe this content is better suited there?